setup.py

```diff
@@ -16,6 +16,9 @@
                   libraries=['sndfile'],
                   sources=['src/accuraterip-checksum.c'])
     ],
+    extras_require={
+        'coverart':  ["pillow"]
```

Comment:
Let’s be consistent:
```suggestion
        'cover_art':  ["pillow"]
```

Reply:
It wasn’t changed… ?

Reply:
I think this is resolved now :D .